osu.Game/Overlays/Direct/DirectListPanel.cs

```diff
@@ -0,0 +1,197 @@
+                        {
+                            AutoSizeAxes = Axes.Both,
+                            Direction = FillDirection.Vertical,
+                            Children = new Drawable[]
+                            {
+                                new OsuSpriteText
+                                {
+                                    Current = localisation.GetUnicodePreference(SetInfo.Metadata.TitleUnicode, SetInfo.Metadata.Title),
+                                    TextSize = 18,
+                                    Font = @"Exo2.0-BoldItalic",
+                                },
+                                new OsuSpriteText
+                                {
+                                    Current = localisation.GetUnicodePreference(SetInfo.Metadata.ArtistUnicode, SetInfo.Metadata.Artist),
+                                    Font = @"Exo2.0-BoldItalic",
+                                },
+                                new FillFlowContainer
+                                {
+                                    AutoSizeAxes = Axes.X,
+                                    Height = 20,
+                                    Margin = new MarginPadding { Top = vertical_padding, Bottom = vertical_padding },
+                                    Children = GetDifficultyIcons(),
+                                },
+                            },
+                        },
+                        new FillFlowContainer
+                        {
+                            Anchor = Anchor.TopRight,
+                            Origin = Anchor.TopRight,
+                            AutoSizeAxes = Axes.Both,
+                            Direction = FillDirection.Vertical,
+                            Margin = new MarginPadding { Right = height - vertical_padding * 2 + vertical_padding },
+                            Children = new Drawable[]
+                            {
+                                new Statistic(FontAwesome.fa_play_circle, SetInfo.Beatmaps.FirstOrDefault()?.OnlineInfo.PlayCount ?? 0)
+                                {
+                                    Margin = new MarginPadding { Right = 1 },
+                                },
+                                new Statistic(FontAwesome.fa_heart, SetInfo.Beatmaps.FirstOrDefault()?.OnlineInfo.FavouriteCount ?? 0),
+                                new FillFlowContainer
+                                {
+                                    Anchor = Anchor.TopRight,
+                                    Origin = Anchor.TopRight,
+                                    AutoSizeAxes = Axes.Both,
+                                    Direction = FillDirection.Horizontal,
+                                    Children = new[]
+                                    {
+                                        new OsuSpriteText
+                                        {
+                                            Text = @"mapped by ",

```

Comment:
we should be using them, so when we get to adding localisation it can be added automatically